substrate/frame/state-trie-migration/src/lib.rs

```diff
@@ -601,7 +653,10 @@ pub mod pallet {
 
 			// ensure they can pay more than the fee.
 			let deposit = T::SignedDepositPerItem::get().saturating_mul(limits.item.into());
```

Comment:
@ggwpez @kianenigma 
The last thing I am not sure about, why the deposit for `continue_migrate` does not include `T::SignedDepositBase`?
Is it a bug or intentional?

```
		/// The base value of [`Config::SignedDepositPerItem`].
		///
		/// Final deposit is `items * SignedDepositPerItem + SignedDepositBase`.
		#[pallet::no_default]
		type SignedDepositBase: Get<BalanceOf<Self>>;
```

The deposits for `migrate_custom_top` and `migrate_custom_child` have it included.

```
/// continue_migrate
let deposit = T::SignedDepositPerItem::get().saturating_mul(limits.item.into());      

/// migrate_custom_top
let deposit = T::SignedDepositBase::get().saturating_add(
        T::SignedDepositPerItem::get().saturating_mul((keys.len() as u32).into()),
);
                        
/// migrate_custom_child  
let deposit = T::SignedDepositBase::get().saturating_add(
        T::SignedDepositPerItem::get().saturating_mul((child_keys.len() as u32).into()),
);
```

Reply:
Yes it is a bug. Since it is not stored anywhere please fix it to match all others. A helper function would be good to prevent a similar one in the future.

Reply:
ok, thank you, fixed here: https://github.com/paritytech/polkadot-sdk/pull/1801/commits/338a494c4b945dbda0aa9cf103dd5218b22d626b